Lecture6(GithubAPI)/hooks/useCities.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from "react"
+
+const useCities = (state, apiData, stateList) => {
+    const [cities, setCities] = useState([]);
+
+    useEffect(() => {
+        fetchCities();
+        async function fetchCities() {
+            const json = await apiData[state];
+            setCities(json);
+        }
+    }, [state, stateList])
```

Comment:
why state and stateList?

Reply:
I was trying to call city once the data is state data is loaded from API eg so that it was be assigned with first city of andaman nicobar but now added select state and city option so don't need will remove it.